name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/csv/VINISExporter.java

```diff
@@ -0,0 +1,174 @@
+        Money buyTotalValue = Money.of(baseCurrency, 0);
+        Money currentTotalValue = Money.of(baseCurrency, 0);
+        Money cash = Money.of(baseCurrency, 0);
+        
+        List<AssetPosition> assets =  performanceToday.getEndClientSnapshot().getAssetPositions()
+                                        .collect(Collectors.toList());
+
+        for (AssetPosition asset : assets)
+        {
+            //calc security values
+            if(asset.getPosition() != null && asset.getSecurity() != null)
+            {
+                if (!asset.getFIFOPurchaseValue().getCurrencyCode().equals(baseCurrency))
+                {
+                    buySecurityValue = buySecurityValue.add(converter.convert(LocalDate.now(), asset.getFIFOPurchaseValue()));
+                    currentSecurityValue = currentSecurityValue.add(converter.convert(LocalDate.now(), asset.getValuation()));
+                }
+                else
+                {
+                    buySecurityValue = buySecurityValue.add(asset.getFIFOPurchaseValue());
+                    currentSecurityValue = currentSecurityValue.add(asset.getValuation());
+                }   
+            }
+            
+            //calc total values
+            if (!asset.getFIFOPurchaseValue().getCurrencyCode().equals(baseCurrency))
+            {
+                //use purchaseValue only for securities, on accounts we only have the valuation 
+                if(asset.getPosition() != null && asset.getSecurity() != null)
+                    buyTotalValue = buyTotalValue.add(converter.convert(LocalDate.now(), asset.getFIFOPurchaseValue()));
+                else
+                    buyTotalValue = buyTotalValue.add(converter.convert(LocalDate.now(), asset.getValuation())); 
+                
+                currentTotalValue = currentTotalValue.add(converter.convert(LocalDate.now(), asset.getValuation()));
+            }
+            else
+            {
+              //use purchaseValue only for securities, on accounts we only have the valuation  
+                if(asset.getPosition() != null && asset.getSecurity() != null)
+                    buyTotalValue = buyTotalValue.add(asset.getFIFOPurchaseValue());
+                else
+                    buyTotalValue = buyTotalValue.add(asset.getValuation());
+                
+                currentTotalValue = currentTotalValue.add(asset.getValuation());
+            }
+            
+        }
+
+        for (AccountSnapshot account : performanceToday.getEndClientSnapshot().getAccounts())
+            cash = cash.add(account.getFunds());   
+        
+        
+        // write to file
+        try (CSVPrinter printer = new CSVPrinter(
+                        new OutputStreamWriter(new FileOutputStream(file), StandardCharsets.UTF_8),
+                        CSVExporter.STRATEGY))
+        {
+            writeHeader(printer);
+            
+            write(printer, "Kontostände Summe", Values.Amount.format(cash.getAmount()));
```

Comment:
Das ist meine einzig relevante Anmerkung:

Muss das Label auf deutsch sein für die VINIS App? Oder kann das auch in der entsprechenden Sprache sein? Ich habe für Teile schon übersetzte Labels, die könnte ich einfügen. Die anderen zumindest übersetzbar machen.

Und zweitens: macht es nicht vielleicht Sinn eine dritte Spalte mit der Währung dem CSV hinzufügen? Auch wenn die VINIS App das aktuell nicht nutzen würde, hätte zumindest die CSV Datei die vollständige Information